README.md

```diff
@@ -1 +1,71 @@
-# Hello-Microverse
\ No newline at end of file
+My first Micronaut Project
+
+> This is my first project as a Microverse student.
+
+![screenshot]()
+
```

Comment:
- [ ] Kindly, fix this issue of screenshot image

Reply:
done